tasks/make-rootfs/run.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+SUFFIX="${STACKS_SUFFIX:-}"
+
+src-ci/concourse/tasks/common/start-docker.sh
```

Comment:
Before I even mention this I realize the insanity of the comment:
Should we not use the concourse docker resource for this? Instead of this script that's caused us issues in the past? And yes, I know that it's the same script. I think that there's benefits in getting later versions from upstream that may solve problems with it before we run into them.